datafusion/sqllogictest/test_files/clickbench.slt

```diff
@@ -23,7 +23,7 @@
 # create.sql came from
 # https://github.com/ClickHouse/ClickBench/blob/8b9e3aa05ea18afa427f14909ddc678b8ef0d5e6/datafusion/create.sql
 # Data file made with DuckDB:
-# COPY (SELECT * FROM 'hits.parquet' LIMIT 10) TO 'clickbench_hits_10.parquet' (FORMAT PARQUET);
+# COPY (SELECT * FROM 'hits.parquet' LIMIT 10) TO 'clickbench_hits_10.parquet' STORED AS PARQUET;
```

Comment:
Here I think we should also maintain backwards compatibility. It is nice to have a dedicated keyword for the often required format option, but I think user's queries in the old syntax should continue working

Reply:
I firmly believe that a singular approach to the format introduction is the most logical and maintainable route. Thus implementing a breaking change aligns better with our goals for clarity and efficiency.

For this particular highlight, I didn't read the DuckDB reminder, this is why I changed it back.

Reply:
I agree that supporting the duckdb style would be nice, but that this PR's consistency is also nice

I wonder if it would be possible (as a follow on PR) to add some special case backwards compatibility code to support the `(FORMAT PARQUET)` style? It seems to me like this could be a small amount of additional, localized code, and then we could going forward use the unified approach in this PR by default?

Reply:
Filed https://github.com/apache/arrow-datafusion/issues/9713 to track